src/math/polynomial/upolynomial_factorization.cpp

```diff
@@ -27,8 +27,6 @@ Module Name:
 #include "math/polynomial/upolynomial_factorization_int.h"
 #include "util/prime_generator.h"
 
-using namespace std;
```

Comment:
this change has nothing to do with the rest of the PR. Please don't mix everything in one go.

Reply:
Sorry, my bad. I thought I had removed that experiment, but I must have accidentally merged that commit with this one.